internal/driver/akamai/agent.go

```diff
@@ -102,7 +104,7 @@ func ExecuteAkamaiAgent() error {
 
 	cache, _ := lru.New[string, int](64)
 
-	akamaiAgent = &AkamaiAgent{
+	akamaiAgent := &AkamaiAgent{
```

Comment:
ok this is a bit strange because there is still the `func Sync(ctx context.Context, req stormrpc.Request)` function that uses the global variable and the rpc handler from line 129-130.
We can refactor this code because the rpc server can invoke the Sync function directly.